code/controllers/subsystem/time_track.dm

```diff
@@ -140,13 +143,19 @@ SUBSYSTEM_DEF(time_track)
 			length(SSair.hotspots),
 			length(SSair.networks),
 			length(SSair.high_pressure_delta),
+			//length(SSair.active_super_conductivity), // LINDA, I assume
```

Comment:
we call this thermal transfer, not superconductivity. Might still exist

Reply:
Is it important that we track it? 

I will add it back if so, I just assumed we didnt really care about it